client/data/user-devices/use-user-devices-query.js

```diff
@@ -0,0 +1,9 @@
+import { useQuery } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+export const useUserDevicesQuery = () =>
```

Comment:
Now that we're overhauling this code, does it make sense to change the naming a bit, to reflect the fact that this is about notifications for the user's devices? Otherwise, I find the current naming a bit confusing.

Reply:
It is a bit long, but works for me since it describes what it's doing 👍

Reply:
Opted for `useNotificationDevicesQuery` because that's the main thing we fetch which _also_ contain settings for each device. Let me know if that also works for you.

Reply:
Let's keep that. Why does naming have to be so hard 😅